bin/lib/zopen-build

```diff
@@ -303,31 +300,26 @@ setEnv()
     export CXXFLAGS="${ZOPEN_CXXFLAGSD} ${ZOPEN_EXTRA_CXXFLAGS}"
   fi
 
+  temp_c_file="/tmp/$LOGNAME.c"
```

Comment:
is it possible to pipe code to the compiler instead? If not, can you check for TMPDIR and TMP too first?

Reply:
Unfortunately, the compiler does not accept source from stdin.  The clang compiler does support it.  But for xlclang, I'll make the updates to check for TMPDIR/TMP